cinnamon-remix/calamares-logs-helper

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
```

Comment:
Let's move this to the common settings instead of having a per-flavor executable.